eng/versioning/version_set_dev.py

```diff
@@ -25,8 +25,7 @@ def format_build_id(build_id):
 
 def get_dev_version(current_version, build_id):
     parsed_version = parse(current_version)
-    #release = parsed_version.release
-    return "{0}.dev{1}".format(parsed_version, build_id)
+    return "{0}a{1}".format(parsed_version.base_version, build_id)
```

Comment:
From this:
![image](https://user-images.githubusercontent.com/45376673/92048139-cad48480-ed3b-11ea-9d7d-3b3221f41a55.png)

to this:
![image](https://user-images.githubusercontent.com/45376673/92048125-c4460d00-ed3b-11ea-8263-512740aca330.png)

Reply:
This is what I intended to do. Is there something I need to fix because of that change?